<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>467</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>949</width>
    <height>467</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>949</width>
    <height>467</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SVM Training Wizard</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Support Vector Machine Training Wizard</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="trainTextbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="trainSelectFile">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSuspicious">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Suspicious</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonNotSuspicious">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Not Suspicious</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainExtractButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainTrainButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trainExitButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="trainProgressWindow">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="trainProgressBar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>80</y>
     <width>20</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>931</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>451</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Please open a file and choose whether it is suspicious or not.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>491</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Click 'Extract' button to create descriptor file from the selected video.</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>931</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>491</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Click 'Train' button to train the SVM with the created descriptor file.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>671</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Training Procedure may take long time to complete. Please be patient.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
